### PR TITLE
Cleanup: delete unused less & more specific selectors

### PR DESCRIPTION
**Pull request by robertkowalski**
- delete not used less
 - make css more specific

### CODE REVIEW

File: assets/less/fauxton.less [hunk 1]
```diff
@@ -185,15 +185,32 @@ table.databases {
   }
 }
 
-.notification-icon {
-  padding-right: 8px;
+.global-notification.alert-success {
```
Comment (garrensmith) [inline assets/less/fauxton.less]: With less you could put all the alert info in the `global-notifications` and less will flatten it for you. Like this:

    .global-notification {
      alert-status {}
      alert-info
    }

**Comment by garrensmith** [comment]
This looks good. I've added a comment on less style. But otherwise I'm +1

**Comment by garrensmith** [comment]
+1 nice clean up.

**Comment by robertkowalski** [comment]
a7a391ebf2f264359b1843985dfe1df382bf898c

**Pull request closed without merging**